### PR TITLE
Release notes for 7.13.4

### PR DESCRIPTION
**Pull request by logstashmachine**
**PREVIEW:** https://logstash_13061.docs-preview.app.elstc.co/guide/en/logstash/current/logstash-7-13-4.html

### CODE REVIEW

**Review by karenzone — APPROVED** [review]
LGTM

**Review by andsel — APPROVED** [review]
LGTM

Re-launched the tool release notes generator (`tools/release/generate_release_notes.rb 7.13 7.13.4`) and nothing changed.

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -47,6 +48,11 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-13-4]]
+=== Logstash 7.13.4 Release Notes
+
+No user-facing changes in Logstash core.
```
Comment (andsel) [inline docs/static/releasenotes.asciidoc:54]:
```suggestion
Fixed bug that sometimes happened when multiple pipelines with GeoIP filter tries to update the local database file. #13072
```

**Review by andsel — CHANGES_REQUESTED** [review]
@karenzone yesterday was merged a bugfix for GeoIP, so it's the only change for `7.13.4`

**Comment by karenzone** [comment]
Thanks for adding the fixed issue, @andsel.  I accepted your suggestion, and added a link to the issue. 
I also added a PREVIEW link in the item description so that you can see how it renders.

**Review by karenzone — APPROVED** [review]
Thanks for adding the fixed issue, @andsel. I accepted your suggestion, and added a link to the issue.  I also added a PREVIEW link in the item description so that you can see how it renders. LGTM!

**Review by andsel — APPROVED** [review]

**Pull request merged**